packages/itwinui-react/src/core/Slider/Slider.tsx

```diff
@@ -412,9 +437,18 @@ export const Slider = React.forwardRef((props, ref) => {
 
     if (Array.isArray(tickLabels)) {
       return (
-        <Box className='iui-slider-ticks'>
+        <Box
+          as='div'
+          {...tickProps}
+          className={cx('iui-slider-ticks', tickProps?.className)}
+        >
           {tickLabels.map((label, index) => (
-            <Box as='span' key={index} className='iui-slider-tick'>
+            <Box
+              as='span'
+              {...tickProps}
+              key={index}
+              className='iui-slider-tick'
+            >
```

Comment:
`iui-slider-ticks` seem to be a wrapper class of `iui-slider-tick` (notice one has "s") so I think they need different props

Reply:
also don't forget to add test for this too